filebeat/harvester/log.go

```diff
@@ -105,6 +106,11 @@ func (h *Harvester) Harvest() {
 		if !h.sendEvent(event) {
 			return
 		}
+
+		if h.Config.CloseTTL > 0 && time.Since(h.startTime) > h.Config.CloseTTL {
```

Comment:
yes, because I don't see any use cases for close_ttl = 0, as this would the harvester directly again.